parmed/modeller/residue.py

```diff
@@ -81,6 +81,7 @@ class ResidueTemplate(object):
     def __init__(self, name=''):
         self.atoms = AtomList()
         self.bonds = TrackedList()
+        self.lonepairs = list() # TODO: Should this be a TrackedList?
```

Comment:
This should not be a `TrackedList`.  The `TrackedList` is only for instances where if you change the list (add or delete an item), it means other information needs to be recomputed.

Basically, if any of the atom, residue, bond, ... etc. arrays in an `AmberParm` change, all of the pointer sections need to be recomputed.